ansible/group_vars/all/defaults.yaml

```diff
@@ -9,7 +9,7 @@ kubernetes_version: "1.29.6"
 kubernetes_major_minor: "{{ (kubernetes_version.split('.') | map('trim'))[:2] | join('.') }}"
 kubernetes_semver: "v{{ kubernetes_version }}"
 
-kubernetes_cni_version: "0.9.1"
+kubernetes_cni_version: "0.8.7"
```

Comment:
why this version is downgraded?

Reply:
this is the version of the package that we publish.

Reply:
@dkoshkin do you know where were we getting `0.9.1` packages in airgap bundle?

Reply:
this is used by flatcar.

rpms are using 0.8.7 because of we are building and publishing them.

ubuntu is using a newer version - i believe 1.3.0 of this CNI package.

we should update the kuberentes_cni as we are here https://github.com/mesosphere/dkp-release/pull/129